sources/packages/web/src/composables/institution/useInstitutionState.ts

```diff
@@ -8,7 +8,14 @@ export function useInstitutionState() {
     () => store.state.institution.institutionState as InstitutionStateForStore,
   );
 
+  const getLocationName = (locationId: number) => {
```

Comment:
As the returned value is parsed from store variable, please use a computed here to ensure the reactive nature. 

![image](https://user-images.githubusercontent.com/54600590/186536350-6a570c18-035f-479d-aba3-07d72dc66076.png)

Reply:
I have a few questions on using computed here:

1. What will the result be like when I use it here and use it again in the component ? The location name is rendered with double quotes. Here is a screen shot of what it looks like.

![image](https://user-images.githubusercontent.com/72251620/186544290-02446d7e-5d18-44fb-9503-35f6a210761a.png).

2. Computed is used for reactive getters / properties. How do you pass a parameter to a computed getter in a composable ? Is it good practice to wrap it in a function ? That is what I did to produce the result above. Here is a screenshot of my code. 

![image](https://user-images.githubusercontent.com/72251620/186544620-a2ddfc17-392a-4279-b4a1-75fea975f558.png)

3. Is a good way around this to apply encapsulation partially by using a setter for the location id in the composable ?

Reply:
My apologies, I did not notice the argument on this composable method. You can ignore my comment on using the computed composable property.